src/modules/Builder/features/SaveAndPublish/SaveAndPublish.utils.ts

```diff
@@ -134,6 +134,7 @@ const getScore = (score: ScoreReport, items: ActivityFormValues['items']) => ({
 const getSection = (section: SectionReport, items: ActivityFormValues['items']) => ({
   ...section,
   ...removeReportsFields(),
+  id: undefined,
```

Comment:
is it ok to remove ids, which were received from the back-end from previously saved report items?